concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -17,7 +17,7 @@ Desrochers-2010-330-15	15	calientaplatos	hot plate			Compound	1.35294117647059	1
 Desrochers-2010-330-16	16	cantamañanas	morning coffee			Compound	5	12	5	
 Desrochers-2010-330-17	17	cascanueces	nutcracker			Compound	5.58823529411765	11	1	
 Desrochers-2010-330-18	18	cascarrabias	grumpy			Compound	5.32352941176471	12	3	
-Desrochers-2010-330-19	19	catacaldos	spirit			Compound	1.42569589165057	10		
+Desrochers-2010-330-19	19	catacaldos	spirit	SPIRIT	53	Compound	1.42569589165057	10		
```

Comment:
I think alcohol or something is meant so I'd unmap.

Reply:
It refers to a nosy person or a person that gets involved in a lot of things. I'd also leave it unmapped.